### PR TITLE
Update installation instructions in README.md

### PR DESCRIPTION
**Pull request by ayojava**
Update on Installation instructions on README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,54 @@
...
-
-* 1) Run database locally
-    mongod or sudo mongod
-    * In case it's not installed, install mongo: https://docs.mongodb.com/v3.2/administration/install-community/
-
-
-* 2) Install server dependencies:
+# Commands to install and run the backend application
+
+* Download and install node version 12.18.0 https://nodejs.org
```
Comment (janakavishwajith) [inline README.md:3]: Let's not be that specific, 12.18.0.. Let's keep it as 12.x.x

File: README.md [hunk 2]
```diff
@@ -1,35 +1,54 @@
...
+  
+* Install “nodemon” by running this command : npm install -g nodemon
+  
+* Download and install MongoDB community Edition , note the connection properties  -     https://docs.mongodb.com/manual/administration/install-community/
+  
+* Download and install MongoDBCompass for the UI based DB Management - https://www.mongodb.com/products/compass
```
Comment (janakavishwajith) [inline README.md:9]: Here also let's not be specific. Let's say 'You can install UI app like mongoDB Compass for easiness of the DB  management' or something like that.

File: README.md [hunk 3]
```diff
@@ -1,35 +1,54 @@
...
+
+* Connect  MongoCompass  to Mongodb using the connection properties earlier during installation of mongoDB, create a new Database  called “kivakodb” , and collection “kivakodb”
+
+* Run database locally  mongod or sudo mongod
+  
+* Clone the project from this repo - https://github.com/Tandem-App-Tuni/Kivako-app-backend.git
+  
+* Open the project in your IDE of choice or go to the root folder of the project in the command line or IDE and run
     ```npm install package.json```
```
Comment (janakavishwajith) [inline README.md:18]: this should be "npm install"

File: README.md [hunk 4]
```diff
@@ -1,35 +1,54 @@
...
     nodemon server.js
     ```
 
-* 4) To handle the SAML authentication during the development instead HAKA login for localhost test purposes, run the container image described in https://medium.com/disney-streaming/setup-a-single-sign-on-saml-test-environment-with-docker-and-nodejs-c53fc1a984c9 .The server configurations are already made, so you just need to follow the *Setup our Identity Provider* section.
+* Alternatively from the root-folder-of-application , run 
+  ```
+    npm run start
+    ```
+
+* To handle the SAML authentication during the development instead HAKA login for localhost test purposes, run the container image described in https://medium.com/disney-streaming/setup-a-single-sign-on-saml-test-environment-with-docker-and-nodejs-c53fc1a984c9 .The server configurations are already made, so you just need to follow the *Setup our Identity Provider* section.
```
Comment (janakavishwajith) [inline README.md:32]: I'm not sure about this. I think it's not supporting HAKA login.

**Review by janakavishwajith — CHANGES_REQUESTED** [review]

**Review by mohsinqadeer — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ayojava) [reply README.md]: this would be updated

File: README.md (hunk 2 above)
Comment (ayojava) [reply README.md]: WIll be updated

File: README.md (hunk 3 above)
Comment (ayojava) [reply README.md]: Will be updated

**Review by EAkkanen — APPROVED** [review]
Text looks ok!

**Pull request merged**